cli/io/config.go

```diff
@@ -70,3 +98,111 @@ func (config *Config) Read() {
 	dec := json.NewDecoder(fileToRead)
 	dec.Decode(&config.Map)
 }
+
+
+// getCredentials reads credentials from .brooklyn_cli data formatted for versions > 0.11.0
+// Note that the password is base64 encoded to avoid json formatting problems
+//{
+//  "credentials": {
+//      "password": "cGFzc3dvcmQ=",
+//      "username": "geoff"
+//  },
+//  "skipSslChecks": false,
+//  "target": "http://geoffs-macbook-pro.local:8081"
+//}
+func (config *Config) getCredentials(target string) (username string, password string, err error) {
+	credentials, found := config.Map[credentialsKey].(string)
+	if !found {
+		err = errors.New("No credentials found for " + target)
+		return
+	}
+
+	if decoded, errd := base64.StdEncoding.DecodeString(credentials); errd != nil {
+		err = errors.New("Could not decode credentials for " + target)
+		return
+	} else {
+		userAndPassword := strings.SplitN(string(decoded), ":", 2)
+		if len(userAndPassword) != 2 {
```

Comment:
Ah, I see. So, in theory, it could be an issue if the user has a ':' in their username- but that is quite unlikely.